src/main/java/nextstep/subway/favorite/application/FavoriteService.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.subway.favorite.application;
+
+import nextstep.subway.auth.domain.LoginMember;
+import nextstep.subway.favorite.domain.Favorite;
+import nextstep.subway.favorite.domain.FavoriteRepository;
+import nextstep.subway.favorite.dto.FavoriteRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import nextstep.subway.favorite.exception.DuplicatedFavoriteException;
+import nextstep.subway.favorite.exception.NotFoundFavoriteException;
+import nextstep.subway.member.application.MemberService;
+import nextstep.subway.member.domain.Member;
+import nextstep.subway.station.application.StationService;
+import nextstep.subway.station.domain.Station;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
+public class FavoriteService {
+    private final FavoriteRepository favoriteRepository;
+    private final StationService stationService;
+    private final MemberService memberService;
+
+    public FavoriteService(FavoriteRepository favoriteRepository, StationService stationService, MemberService memberService) {
+        this.favoriteRepository = favoriteRepository;
+        this.stationService = stationService;
+        this.memberService = memberService;
+    }
+
+    public FavoriteResponse createFavorite(LoginMember loginMember, FavoriteRequest request) {
+        final Station source = stationService.findById(request.getSource());
+        final Station target = stationService.findById(request.getTarget());
+        final Member member = memberService.findById(loginMember.getId());
+        if(favoriteRepository.existsByMemberAndSourceAndTarget(member, source, target)){
```

Comment:
중복으로 했을 경우 예외를 발생시키는 것도 좋은데 
중복일 경우 그냥 무시하고 기존의 값을 응답하고 없을 경우엔 새로 `save` 되도록 하는 것도 
하나의 방법이 될 수 있을 것 같습니다 😄 
이 부분도 구조적으로 드린 의견이니 참고만 해주시면 감사하겠습니다 🙇 
```suggestion
        if (favoriteRepository.existsByMemberAndSourceAndTarget(member, source, target)){
```